plans/upgrade.pp

```diff
@@ -445,5 +445,15 @@
 
   peadm::check_version_and_known_hosts($current_pe_version, $_version, $r10k_known_hosts)
 
+    # Log parameters for debugging
```

Comment:
Again, looks good, just move up to be the first thing we do in the plan. Also, would be good to add this to the migrate plan as well